### DISCUSSION TITLE
Where is the Microsoft.Azure.Cosmos.Table codebase?

### DISCUSSION THREAD

**Discussion by MunicipalSoftwareGroup** [Azure/azure-cosmos-dotnet-v3#2625] [Q&A] [answered]
I need to look at the code for the Table API .NET SDK, but the only repo which matches this description (and is linked in NuGet) is a [stale and empty repository](https://github.com/Azure/azure-cosmos-table-dotnet). I can't find a "Table" namespace in this repo so I assume it's somewhere else?

**Comment by kirankumarkolli** [comment, marked as answer]
Table SDK code is not open sourced so not available on GitHub.
